.github/workflows/bad-workflow.yml

```
@@ -0,0 +1,39 @@
+name: Bad Workflow Example
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
```

actionlint (1.7.10)

could not parse as YAML: did not find expected key

(syntax-check)